src/libraries/System.Runtime.Intrinsics.Experimental/ref/System.Runtime.Intrinsics.Experimental.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
@ericstj, @ViktorHofer  who is the right person to review the csproj changes here?

This is an experimental package that we've shipped in the past and which we'll be shipping again. Given its experimental, we shouldn't be concerned about what was contained in the version shipped in past .NET Core versions.

There is also the issue that `AvxVnni` needs to inherit from `Avx2` in S.P.Corelib, but the constructor needs to remain `internal`. Are IVTs the right thing here or do we have something better we can do?

Reply:
I thought we agreed on having experimental packages in runtimelab and not in runtime? cc @jkotas

Reply:
The changes in general LGTM. I'll let @ericstj speak to the IVT question.

Reply:
This is something that is experimental in that sense. Its something we will ship in preview form alongside .NET 6, its just "experimental" in that there isn't hardware for it yet and so testing it and giving it the "stable" stamp isn't easy

Reply:
I see, thanks for the clarification. What I'm still surprised about is why we moved the sources into runtimelab in the first place then?

Reply:
The difference between the experimental runtimelab and preview features is covered here: https://github.com/dotnet/designs/blob/main/accepted/2021/preview-features/preview-features.md#why-did-we-use-the-term-preview

TLDR: 

runtimelab experiments are start of the innovation pipeline. The feature is very unbaked, we expect to interate on the shape a lot, and we do not have high confidence when or whether it is eventually going to ship.

Preview features are end of the innovation pipeline. We are 90%+ there, reasonably confident that the feature is good and going to ship, just need more work or feedback to get it to 100% quality.

Reply:
Thanks for the clarification Jan and Tanner.